src/components/claiming/UnlockWallet.vue

```diff
@@ -39,7 +45,11 @@ export default {
       this.sent = true
     },
     updateView() {
-      this.$router.push('/claiming/download-file')
+      if (localStorage.getItem('completed')) {
+        this.$router.push('/claiming/countdown')
+      } else {
+        this.$router.push('/claiming/download-file')
+      }
```

Comment:
I think we always have to navigate to `/claiming/countdown`
```suggestion
    this.$router.push('/claiming/countdown')
```